actions/release/1.0/internal/conf/conf.go

```diff
@@ -86,13 +86,14 @@ func (c Conf) GetAppName() string {
 }
 
 type Service struct {
-	Name             string
-	Cmd              string   `json:"cmd"`
-	Image            string   `json:"image"`
-	Cps              []string `json:"copys"`
-	RetagImage       string   `json:"retagImage"`
-	RegistryUsername string   `json:"registryUsername"`
-	RegistryPassword string   `json:"registryPassword"`
+	Name               string
+	Cmd                string   `json:"cmd"`
+	Image              string   `json:"image"`
+	Cps                []string `json:"copys"`
+	RetagImage         string   `json:"retagImage"`
+	RegistryUsername   string   `json:"registryUsername"`
+	RegistryPassword   string   `json:"registryPassword"`
+	ReleaseImgNotRetag bool     `json:"releaseImgNotRetag"` // 是否使用RetagImage对应的镜像仓库,默认false为使用RetagReg的镜像仓库,true为同时上传两个镜像仓库【默认镜像】和【Retag镜像】，
```

Comment:
useRetagImage

Reply:
change `ReleaseImgNotRetag ` tag name to `useRetagImage`